pandas/core/arrays/boolean.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+
+from pandas.core.arrays.integer import IntegerArray, _IntegerDtype
+from pandas.core.dtypes.dtypes import registry
+from pandas.util._decorators import cache_readonly
+
+
+class BooleanDtype(_IntegerDtype):
+    name = "EABool"
+    type = np.bool
+    na_value = np.nan
+
+    @cache_readonly
+    def is_signed_integer(self):
+        return False
+
+    @cache_readonly
+    def is_unsigned_integer(self):
+        return False
```

Comment:
For what it's worth, you should be able to get `is_signed_integer` and `is_unsigned_integer` for free without needing to override since `np.dtype(np.bool).kind` returns `'b'`.  This does save performing a single comparison and is more explicit though.